clients/drcachesim/common/trace_entry.h

```diff
@@ -191,6 +191,35 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_CPU_ID,
 
+    /**
+     * The marker value contains function id defined by the user in droption.
```

Comment:
grammar: s/function/the function/

The user doesn't know what "in droption" means: s/droption/the -record_function option/

Reply:
Done